### PR TITLE
Override service restart command

### PR DESCRIPTION
**Pull request by Wiston999**

### CODE REVIEW

**Review by dannyk81 — APPROVED** [review]
superb :+1:

**Comment by dannyk81** [comment]
@Wiston999 I think this can be merged and perhaps we can do a PR upstream? if accepted we won't need to keep a fork.

WDYT?

**Comment by Wiston999** [comment]
Yes, I already created a PR to upstream so I guess we can wait till it is merged

**Comment by dannyk81** [comment]
> Yes, I already created a PR to upstream so I guess we can wait till it is merged

Ohh! indeed https://github.com/voxpupuli/puppet-squid/pull/127

Great :+1:

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
```
Comment (kamunas) [inline .editorconfig:1]: I'd remove this file if we want to get the PR integrated

**Comment by Wiston999** [comment]
Closing this PR as It was already merged in original repository.

**Pull request closed without merging**